modules/platform/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/BatchUploadObject.java

```diff
@@ -434,6 +436,27 @@ public Response getBatchExtraInfo(@PathParam(REQUEST_BATCH_ID) String batchId) t
         return buildResponse(Status.OK, result);
     }
 
+    /** @since 11.1 */
+    @POST
+    @Path("{batchId}/refresh")
```

Comment:
I'd call it `/refreshToken` to be more explicit. "refresh" in the context of a batch upload does not immediately evoke the auth token.